h/hey-listen/hey-listen_ubi_9.3.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package	    : hey-listen
+# Version	    : master
+# Source repo	    : https://github.com/mattgperry/hey-listen
+# Tested on	    : UBI 9.3
+# Language          : Node
+# Travis-Check      : True
+# Script License    : MIT License
+# Maintainer	    : Prachi Kurade <prachi.kurade1@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=hey-listen
+PACKAGE_VERSION=${1:-master}
```

Comment:
Instead of passing master as package version you can use commit id which you verified. That will be more specific.

Reply:
Done.